snapshot/overlay/overlay.go

```diff
@@ -114,7 +114,18 @@ func (o *Snapshotter) Prepare(key, parent string) ([]containerd.Mount, error) {
 }
 
 func (o *Snapshotter) View(key, parent string) ([]containerd.Mount, error) {
-	panic("not implemented")
+	if parent == "" {
```

Comment:
I don't think it makes much sense to view an empty directory. The callers will need to check but I think it makes more sense for the caller to handle it than have the snapshot drivers create an empty directory just to create a ro bind mount to it.